client/post-editor/media-modal/gallery/edit.jsx

```diff
@@ -47,17 +49,28 @@ export default React.createClass( {
 		}
 
 		return (
-			<SortableList onChange={ this.onOrderChanged }>
-				{ settings.items.map( ( item ) => {
-					return (
-						<EditorMediaModalGalleryEditItem
-							key={ item.ID }
-							site={ site }
-							item={ item }
-							showRemoveButton={ settings.items.length > 1 } />
-					);
-				} ) }
-			</SortableList>
+			<div>
+				<SectionHeader>
+					<EllipsisMenu position="bottom right">
+						<PopoverMenuItem onClick={ this.props.onReverse }>
+							{ this.props.translate( 'Reverse Order' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **63** times:
`translate( 'Reverse order' )` *ES Score: 19*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).